docstore/doc.go

```diff
@@ -101,13 +101,12 @@
 // documents. Using structs is preferred because it enforces some structure on your
 // data.
 //
-// Docstore mimics the encoding/json package in its treatment of
-// structs: by default, a struct's exported fields are the fields of the document.
-// You can alter this default mapping by using a struct tag beginning with
-// "docstore:". Docstore struct tags support renaming, omitting fields
+// Docstore treats a struct's exported fields as the fields of the document by
+// default. You can alter this default mapping by using a struct tag beginning
+// with "docstore:". Docstore struct tags support renaming, omitting fields
 // unconditionally, or omitting them only when they are empty, exactly like
-// encoding/json. Docstore also honors a "json" struct tag if there is no "docstore"
```

Comment:
So does docstore not honor `json:"..."`?

Reply:
Not any more, that was changed in https://github.com/google/go-cloud/commit/384e3e9eb5ccdf4a539e6bbf60e797b8795c40d3